lib/Traits/TraitsClasses/GeoJsonTraits.ts

```diff
@@ -55,14 +55,23 @@ export class GeoJsonTraits extends mixTraits(
     type: "boolean",
     name: "Clamp to Ground",
     description:
-      "Whether the features in this GeoJSON should be clamped to the terrain surface."
+      "Whether the features in this GeoJSON should be clamped to the terrain surface. If `forceCesiumPrimitives is false, this will be `true`"
```

Comment:
```suggestion
      "Whether the features in this GeoJSON should be clamped to the terrain surface. If `forceCesiumPrimitives` is false, this will be `true`"
```